### DISCUSSION TITLE
感心するほど凄い人類

### DISCUSSION THREAD

**Discussion by ryuujo1573** [anharu2394/tapioca#65] [General]
こちらは中国の大学3年生で、あんはるさんに比べられる少年（少女？）のことを聞いたことがあっても少ないです。
俺も頑張らないとwwww
英語上手！

**Comment by anharu2394** [comment]
ありがとうございます！(Thank you very much)

**Reply by ryuujo1573** [reply]
これが人間の参差錯落かなって常に思う
あんはるさんに勉強しまっせ
向你学习！
